src/Gateways/Helper/Helper.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace GlobalPayments\WooCommercePaymentGatewayProvider\Gateways\Helper;
+
+defined( 'ABSPATH' ) || exit;
+
+class Helper {
+
+    public static function removeSlashesFromToken( string $token ): string {

```

Comment:
Naming convention, alignment and spaces.

Reply:
done